plugins/API/RowEvolution.php

```diff
@@ -63,9 +63,14 @@ public function getRowEvolution($idSite, $period, $date, $apiModule, $apiAction,
 
         // if goal metrics should be shown, we replace the metrics
         if ($showGoalMetricsForGoal !== false) {
-            $metadata['metrics'] = [
-                'nb_visits' => $metadata['metrics']['nb_visits'],
-            ];
+            if (array_key_exists('nb_visits', $metadata['metrics'])) {
+                $metadata['metrics'] = [
+                    'nb_visits' => $metadata['metrics']['nb_visits'],
+                ];
+            } else {
+                // if no visits are available, simply use the first available metric
+                $metadata['metrics'] = array_slice($metadata['metrics'], 0, 1);
```

Comment:
Should we add a simple screenshot (e.g. in `RowEvolution_spec.js`) that ensures this does not regress?

Reply:
I've added a test for that.